amplify.yml

```diff
@@ -1,42 +1,42 @@
-# Rewrites and redirect configuration
-# Source address  </^[^.]+$|\.(?!(css|html|gif|ico|jpg|js|png|txt|svg|woff|ttf|map|json)$)([^.]+$)/>
-# Target address  /index.html
-# Type            200 (Rewrite)
-version: 2.0
-frontend:
-    phases:
-        preBuild:
-            commands:
-                - echo "Build env:" $BUILD_ENV
-                - npm i
-        build:
-            commands:
-                # Kommunicate Test
-                - if [ "${BUILD_ENV}" = "test" ]; then npm run build-test; fi
-                # Kommunicate Release
-                - if [ "${BUILD_ENV}" = "release" ]; then npm run build-release; fi
-                # Kommunicate Staging
```

Comment:
Check formatting, there should not be any changes in this file

Reply:
Have Checked, Formatting is same